mpdm/Peds[] steps
std_msgs/Float32 cost
std_msgs/String policy
std_msgs/Float32 propagation_time